streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -68,6 +70,17 @@ public void markChangelogAsCorrupted(final Collection<TopicPartition> partitions
         stateMgr.markChangelogAsCorrupted(partitions);
     }
 
+    @Override
+    public void recordProcessLatency(final long elapsedLatencyMs) {
```

Comment:
prop: I find `processLatency` and `totalLatency` a bit misleading. What about `processTime` and `totalProcessTime`.